app/bundles/CoreBundle/Assets/js/4.builder.js

```diff
@@ -351,6 +351,7 @@ Mautic.closeBuilder = function(model) {
             // Trigger slot:destroy event
             document.getElementById('builder-template-content').contentWindow.Mautic.destroySlots();
 
+            var xs = new XMLSerializer();
```

Comment:
Can we use XMLSerializer when no browser supports it? https://developer.mozilla.org/en-US/docs/Web/API/XMLSerializer#Browser_compatibility